### ISSUE TITLE
IfcStore.path is initialised to cwd

### ISSUE THREAD

**Issue by brunopostle** [IfcOpenShell/IfcOpenShell#3065]
Once an IFC project has been saved, `IfcStore.path` is set to the full path to the file:

    >>> import blenderbim.bim.ifc
    >>> blenderbim.bim.ifc.IfcStore.path
    '/tmp/untitled.ifc'

..but when no project is saved it is set to the cwd:

    >>> import blenderbim.bim.ifc
    >>> blenderbim.bim.ifc.IfcStore.path
    '/home/bruno'

I can't figure-out why this is, expectation is that is is set to an empty string.

**Comment by brunopostle** [comment]
The code looks like it was intended to default to "": https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.7.0/src/blenderbim/blenderbim/bim/ifc.py#L32

This cascades down to `tool.Ifc.get_path()`, ie. we should be able to rely on `if tool.Ifc.get_path():` as a check for the project being saved to disk.

**Comment by Moult** [comment]
Agreed, it should probably be either empty string or None. Would you like to submit a PR?

**Comment by brunopostle** [comment]
I couldn't figure out how it was happening, when I ran the suspect code outside blender it did the right thing

**Comment by brunopostle** [comment]
This bug seems to have vanished, path for an unsaved model is an empty string:

```
>>> import bpy
>>> bpy.context.scene.BIMProperties.ifc_file
''
>>> import bonsai.tool
>>> bonsai.tool.ifc.IfcStore.path
''
>>> bonsai.tool.ifc.Ifc.get_path()
''
```

Closing.
